examples/evm/blockExtraDataDecoder.ts

```diff
@@ -0,0 +1,250 @@
+  }
+
+  let displayImportTx = () => {
+    //importTX
+    let importedTxImpInputs = txToObject.unsignedTx.transaction.importIns.map(
+      (inp) => ({
+        Type: inp._typeName,
+        TransactionId: base58Encoder(inp.txid.data),
+        OutputId: toHexThenDec(inp.outputidx.data),
+        AssetID: base58Encoder(inp.assetID.data),
+        Input: {
+          Type: inp.input._typeName,
+          TypeID: inp.input._typeID,
+          SignaturesIds: inp.input.sigIdxs.map((signature) => ({
+            Type: signature._typeName,
+            Source: bufToHex(signature.source),
+            Bytes: bufToHex(signature.bytes.data),
+            BytesSize: signature.bsize
+          })),
+          Amount: bufToHex(inp.input.amount),
+          AmountValue: "0x" + inp.input.amountValue,
+          DecimalAmountValue: fromHexToDec(inp.input.amountValue) + " AVAX"
+        }
+      })
+    )
+    let importedTxOutputs = txToObject.unsignedTx.transaction.outs.map(
+      (out) => ({
+        Address: bufToHex(out.address.data),
+        Amount: bufToHex(out.amount.data),
+        AmountValue: "0x" + out.amountValue,
+        DecimalAmountValue: fromHexToDec(out.amountValue) + " AVAX",
+        AssetID: base58Encoder(out.assetID.data)
+      })
+    )
+    let importedTxCredentials = txToObject.credentials.map((credential) => ({
+      Type: credential._typeName,
+      TypeID: credential._typeID,
+      Signatures: credential.sigArray.map((signature) => ({
+        Type: signature._typeName,
+        Bytes: bufToHex(signature.bytes.data),
+        BytesSize: signature.bsize
+      }))
+    }))
+    let importTx = {
+      Type: txToObject._typeName,
+      UnsignedTx: {
+        Type: txToObject.unsignedTx._typeName,
+        CodecID: txToObject.unsignedTx.codecID,
+        Transaction: {
+          Type: txToObject.unsignedTx.transaction._typeName,
+          TypeID: txToObject.unsignedTx.transaction._typeID,
+          NetworkID: toHexThenDec(
+            txToObject.unsignedTx.transaction.networkID.data
+          ),
+          BlockchainID: base58Encoder(
+            txToObject.unsignedTx.transaction.blockchainID.data
+          ),
+          BlockchainIDName: chainName(
+            txToObject.unsignedTx.transaction.blockchainID.data
+          ),
+          SourceChain: base58Encoder(
+            txToObject.unsignedTx.transaction.sourceChain.data
+          ),
+          SourceChainName: chainName(
+            txToObject.unsignedTx.transaction.sourceChain.data
+          ),
+          NumberOfImportedInputs: toHexThenDec(
+            txToObject.unsignedTx.transaction.numIns.data
+          ),
+          ImportedInputs: importedTxImpInputs,
+          NumberOfOutputs: toHexThenDec(
+            txToObject.unsignedTx.transaction.numOuts.data
+          ),
+          Outputs: importedTxOutputs
+        }
+      },
+      Credentials: importedTxCredentials
+    }
+    console.log(require("util").inspect(importTx, true, 10))
```

Comment:
probs need to remove this

Reply:
It's part of the output
```
Credentials: [
    {
      Type: 'SECPCredential',
      TypeID: 9,
      Signatures: [
        {
          Type: 'Signature',
          Bytes: '0xf526c9a38a2da08291583bf86e5160bd8b49df585b3fc2fb57884390c673f748428c58e95c6514b9d6a27d273550c63070ab64d257798e8d07f8a208489ebb2100',
          BytesSize: 65
        },
        [length]: 1
      ]
    },
    [length]: 1
  ]
```
}

Should I still remove it?